homeassistant/components/pi_hole/sensor.py

```diff
@@ -48,6 +41,25 @@ def name(self):
         """Return the name of the sensor."""
         return f"{self._name} {self._condition_name}"
 
+    @property
+    def server_unique_id(self):
+        """Return the unique id of the server."""
+        return f"{self.pi_hole.api.host}/{self.pi_hole.api.location}"
```

Comment:
https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements
`host` shouldn't be used for a unique_id.  If there isn't a good choice, use the config entry entry_id instead

Reply:
What about the config flow unique id? Should I just remove that?

Reply:
If there isn't a stable source for the unique id, yes best to remove.